features/binlog.feature

```diff
@@ -51,3 +51,94 @@ Feature: Binlog
 		And my-peer becomes disconnected
 		And I send a passive check result for service PONG on host hostA with 2 MiB output data
 		Then file merlin.log matches WARNING: Maximum binlog size reached for node my-peer
+
+	Scenario: Binlog is saved on disk for persistence if nodes are
+		disconnected and we restart this nodes.
+		Given I start naemon with merlin nodes connected
+			| type   | name    | port | data_timeout |
+			| peer   | my-peer | 4001 | 1            |
+		And I wait for 2 seconds
+
+		When my-peer becomes disconnected
+		And I send naemon command PROCESS_HOST_CHECK_RESULT;hostA;1;Fromtest
+		And I send naemon command SHUTDOWN_PROGRAM
+		And I wait for 2 seconds
+		Then the file module.my-peer.binlog.save should exist
+		And the file module.my-peer.binlog.meta should exist
+
+	Scenario: Binlog is not saved on disk when nodes are connected
+		Given I start naemon with merlin nodes connected
+			| type   | name    | port |
+			| peer   | my-peer | 4001 |
+		And I wait for 1 seconds
```

Comment:
It's mostly waiting for (actual merlin) to connect to the cukemerlin nodes (sending ctrl actives back and forth) rather than waiting for Naemon.

I replaced it in the steps possible, however in some steps we're waiting for the nodes to be disconnected as per the `data_timeout`